RoaringBitmap/src/test/java/org/roaringbitmap/RangeBitmapTest.java

```diff
@@ -69,7 +69,7 @@ public void testInsertReversedContiguousValues(int size) {
   @ValueSource(ints = {0, 0xFFFF, 0x10001, 100_000, 0x110001, 1_000_000})
   public void testLessThanZeroEmpty(int size) {
     RangeBitmap.Appender appender = RangeBitmap.appender(size);
-    LongStream.range(0, 10).forEach(appender::add);
+    LongStream.range(0, Math.min(size, 10)).forEach(appender::add);
```

Comment:
had to change this because if the max value was 0 before, it would accept values up to 255